vnext/Desktop.UnitTests/MemoryMappedBufferTests.cpp

```diff
@@ -0,0 +1,149 @@
+#include "MemoryMappedBuffer.h"
+#include "utilities.h"
+
+#include <CppUnitTest.h>
+
+#include <shlwapi.h>
+#include <windows.h>
+
+#include <cstring>
+#include <memory>
+
+using facebook::jsi::Buffer;
+using facebook::jsi::JSINativeException;
+using facebook::react::utilities::checkedReinterpretCast;
+using Microsoft::JSI::makeMemoryMappedBuffer;
+using Microsoft::VisualStudio::CppUnitTestFramework::Assert;
+
+namespace {
+
+static constexpr const wchar_t *const testFileName = L"temp.txt";
```

Comment:
>t t [](start = 36, length = 3)

TestFileName or s_testFileName or g_testFileName

Reply:
Fixed.

---
In reply to: [320918079](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320918079) [](ancestors = 320918079)